docs/contributing.md

```diff
@@ -71,6 +71,9 @@ Most of our style is pretty easy to pick up on, but right now it's not entirely
   * If you not sure if a comment is obvious, go ahead and include it.
 * In general we don't wrap lines, they can be as long as needed. If you do choose to wrap lines please do not wrap any wider than 76 columns.
 * We use `#pragma once` at the start of header files rather than old-style include guards (`#ifndef THIS_FILE_H`, `#define THIS_FILE_H`, ..., `#endif`)
+* We accept both forms of pre-processor if's: `#ifdef DEFINED` and `#if defined(DEFINED)`
```

Comment:
```suggestion
* We accept both forms of preprocessor if's: `#ifdef DEFINED` and `#if defined(DEFINED)`
```
I think “preprocessor” is the common form. I haven't seen it spelled “pre-processor” before.